packages/alea-frontend/pages/api/blog/create-post.ts

```diff
@@ -14,11 +14,30 @@ export default async function handler(
   if (!isModerator(userId)) {
     return res.status(403).send({ message: 'Unauthorized.' });
   }
-  const { title, body, postId, authorId, authorName } = req.body;
+  const { title, body, postId, authorId, authorName, heroImageId, heroImageUrl } = req.body;
+
+  let sqlQuery = `INSERT INTO BlogPosts (title, body, postId, authorId, authorName`
+  let addedQuery = `) VALUES (?, ?, ?, ?, ?`;
```

Comment:
why complicate things? just let it be set to`null` when !heroimageId or !heeoImageUrl